specification/dimensions/chargeperiodend.md

```diff
@@ -0,0 +1,32 @@
+# Charge Period End
+
+Charge period represents the time window in which a charge was incurred. The time window is inclusive of the charge period start date and exclusive of the charge period end date. A charge can start and/or end at any time within a charge period window and can span multiple charge periods.
```

Comment:
Given "charge period" isn't a consistent, defined time frame, we probably don't need this last part. I'm not sure what it would mean to "span multiple charge periods".
```suggestion
Charge period represents the time window in which a charge was incurred. The time window is inclusive of the charge period start date and exclusive of the charge period end date. A charge can start and/or end at any time within a charge period window.
```

Reply:
Updated to "The charge period for continuous usage should match the time granularity of the dataset (e.g., 1 hour for hourly, 1 day for daily)."